src/c++/perf_analyzer/genai-perf/tests/test_profile_data_parser.py

```diff
@@ -78,14 +81,53 @@ def write(self: Any, content: str) -> int:
 
         return written_data
 
-    def test_base_profile_data(self, mock_read_write: pytest.MonkeyPatch) -> None:
+    embedding_profile_data = {
```

Comment:
Would you be able to add big comment headers to separate the 2-3 sections of this test? Something like [this](https://github.com/triton-inference-server/server/blob/c61d993ff3e3608d9b76e9c89a65e5ce41497d49/qa/L0_grpc_state_cleanup/cleanup_test.py#L405-L407).

Reply:
Sure, addeded comment header

Reply:
Looks great!